client/src/components/AssignPersonModal.js

```diff
@@ -44,15 +44,19 @@ const AssignPersonModal = ({ position, showModal, onCancel, onSuccess }) => {
   const [error, setError] = useState(null)
   const [person, setPerson] = useState(position && position.person)
   const [doSave, setDoSave] = useState(false)
+  const [removeUser, setRemoveUser] = useState(false)
 
   const save = useCallback(() => {
     let graphql, variables
     if (person === null) {
+      !isAdvisor() && Position.downgradePermission(latestPositionProp.current)
```

Comment:
Better written as an if-condition [ `if (!isAdvisor()) { … }` ]; this looks like a logical expression with side-effects (i.e. would violate https://eslint.org/docs/rules/no-unused-expressions ).